### PR TITLE
SC-19045 add agent events

### PR DESCRIPTION
**Pull request by cpipilas**
- Add agent events page under Sematext Agent
- Update events page with Agent events section
- Update event examples page
- Move container events under agent events
- Update Integrations > Kubernetes page
- Update Monitoring > containers
- Renamed Agent > Kubernetes > Events to Event Configuration
- Update left nav

### CODE REVIEW

**Review by yago-123 — APPROVED** [review]
lgtm

File: docs/agents/sematext-agent/events.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Agent stopped
+
+This event is sent when the agent is stopped.
+
+Message: agent_stopped sematext-agent `version` / on `hostname`
+
+## Out of memory (OOM)
+
+This event is sent when a process is terminated by OOM Killer.
```
Comment (otisg) [inline docs/agents/sematext-agent/events.md:19]: Maybe we want to explicitly say that this is about any Linux process, not agent process

Comment (otisg) [reply docs/agents/sematext-agent/events.md:19]: I see (Linux only) below.  If this is Linux only as well, we should add that.

File: docs/agents/sematext-agent/events.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+title: Sematext Agents Events
+
+Below you can find a list of all the events sent from Sematext Agent
```
Comment (otisg) [inline docs/agents/sematext-agent/events.md:3]: Should we provide link(s) to where in SC Infra one can see these events?
Should we point out Events heatbar in Monitoring?
Should we mention Split Screen with Events view on one side?

File: docs/agents/sematext-agent/events.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Package removed
+
+This event is sent when a new package is removed (Linux only).
+
+Message: Package `package name`, version `version` has been removed on host `hostname`
+
+Package types supported: Node, Python 2.x/3.x, Deb, RPM
+
+## Docker events
```
Comment (otisg) [inline docs/agents/sematext-agent/events.md:39]: @yago-123 Docker or Container?
@cpipilas we seem to be using Docker and Container interchangeably.  Tag names are container.xxxx.

File: docs/agents/sematext-agent/events.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+Sematext Agent sends supports all events from various docker object types, including containers, images, plugins, volumes, networks, daemons, services, nodes, secrets and configs. Learn more about all the available docker events at: https://docs.docker.com/reference/cli/docker/system/events/.
```
Comment (otisg) [inline docs/agents/sematext-agent/events.md:41]: sends supports ==> collects?

**Review by otisg — CHANGES_REQUESTED** [review]

File: docs/agents/sematext-agent/events.md (hunk 1 above)
Comment (cpipilas) [reply docs/agents/sematext-agent/events.md]: added Linux Events section.

File: docs/agents/sematext-agent/events.md (hunk 3 above)
Comment (cpipilas) [reply docs/agents/sematext-agent/events.md]: Add Container events section and Docker events subsection

File: docs/events/event-examples.md [hunk 5]
```diff
@@ -7,11 +7,11 @@ Events originate from internal and external sources.
 
 Internal events come from sources such as:
 
-* [Kubernetes events](https://sematext.com/docs/integration/kubernetes/#kubernetes-events) collected by Sematext Agent
-* [Container events](https://sematext.com/docs/monitoring/containers/#container-events) collected by Sematext Agent
+* [Kubernetes events](https://sematext.com/docs/agents/sematext-agent/events/#kubernetes-events) collected by Sematext Agent
+* [Container events](https://sematext.com/docs/agents/sematext-agent/events/#container-events) collected by Sematext Agent
+* [Linux events](https://sematext.com/docs/agents/sematext-agent/events/#linux-events) like OOM Killer events
+* [Events internal to Sematext Agent](https://sematext.com/docs/agents/sematext-agent/events/#internal-events-from-sematext-agent), such as agent starting and stopping
```
Comment (otisg) [inline docs/events/event-examples.md:13]: Events internal to Sematext Agent ==> Sematext Agent events

**Review by otisg — APPROVED** [review]
LGTM. Just one small change.

**Pull request merged**